ESP8266/ESP8266.h

```diff
@@ -218,6 +219,7 @@ class ESP8266
 private:
     UARTSerial _serial;
     ATCmdParser _parser;
+    Mutex _smutex; // One at a time must access the serial line
```

Comment:
"Must" is a bit wrong, its more like "only one should access the ..."
Change it to something like "// Protect serial port access"